source/iofs/README.md

```diff
@@ -0,0 +1,38 @@
+# iofs
+
+Driver with file system interface (`io/fs#FS`) supported from Go 1.16.
+
+This Driver cannot be used with Go versions 1.15 and below.
+
+## Usage
+
+Directory embedding example
+
+```go
+package main
+
+import (
+	"embed"
+	"log"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+)
+
+//go:embed migrations
```

Comment:
```suggestion
//go:embed migrations/*.sql
```

May be useful to describe this to avoid people from accidentally embedding their own `README.md`, `.gitignore`, `.DS_Store`, etc.